ssm/invocation/runner_test.go

```diff
@@ -36,7 +36,7 @@ func TestRunSSMCommand(t *testing.T) {
 	})
 
 	t.Run("dry run flag true", func(t *testing.T) {
-		go RunSSMCommand(mockSvc, mockParams, true, resultChan, errChan, "i-123456", "i-654321")
+		go RunSSMCommand(mockSvc, mockParams, "1", "0", true, resultChan, errChan, "i-123456", "i-654321")
```

Comment:
Any tests for the use of percent? like 10%?

Reply:
additional tests added. Also added testing for cmdutil/helpers (specifically for ValidateIntOrPercantageValue)